src/BinSkim.Driver/AnalyzeOptions.cs

```diff
@@ -43,5 +43,15 @@ public class AnalyzeOptions : AnalyzeOptionsBase
             HelpText = "Emit verbose output. The resulting comprehensive report is designed to provide appropriate evidence for compliance scenarios.")]
         [Obsolete("Use --level and --kind instead.")]
         public bool Verbose { get; set; }
+
+        [Option(
+            "repository-uri",
+            HelpText = "RepositoryUri is required to rule BA4001 and BA4002.")]
+        public string RepositoryUri { get; set; }
+
+        [Option(
+            "pipeline-name",
+            HelpText = "PipelineName is required to rule BA4001 and BA4002.")]
```

Comment:
>PipelineName [](http://example.com/codeflow?start=24&length=12)

same note, what is this data? you don't describe it.

Reply:
updated the helptext, pls, review :)